README.md

```diff
@@ -12,17 +12,42 @@ This library provides a higher-level API to
 
 See the `Proto3.Suite.Tutorial` module for more details.
 
-# Running the language interop tests
+## Running the language interop tests
 
 We test inter-language interop using protoc's built-in Python code generation. In
 order to successfully run these tests, you'll need to install the google protobuf
 Python library. It's best to create a virtualenv and then use pip to install the
 right version (virtualenv is a python utility which can be installed with pip).
 
-```
+```bash
 $ virtualenv pyenv
 $ source pyenv/bin/activate
 $ pip install protobuf==3.0.0b3  # Need the latest version for the newest protoc
 ```
 
 `brew install python` may also work.
+
+Alternate, the `nix-shell` environment provides a complete incremental build
+and test environment. From the root fo this repository:
+
+```bash
+$ nix-shell release.nix -A proto3-suite.env
+[nix-shell]$ cabal configure --with-gcc=clang --enable-tests
+[nix-shell]$ cabal build
+[nix-shell]$ cabal test
+```
+
+## `compile-proto-file`
+
+Run the following commmand from the root of this repository to install the
+`compile-proto-file` executable:
+
+```bash
+$ nix-env -iA proto3-suite -f release.nix
```

Comment:
I'd suggest using the longer `--install --attr` flags so that people less familiar with Nix can more easily guess what this command does before running it